src/Shell/WorkerShell.php

```diff
@@ -44,11 +47,28 @@ public function main()
             });
         }
 
-        while($worker->work()) {
+        while ($worker->work()) {
             if ($worker->returnCode() != GEARMAN_SUCCESS) {
                 echo "return_code: " . $worker->returnCode() . "\n";
                 break;
             }
         }
     }
+
+    /**
+     * Returns an array with all configured jobs.
+     *
+     * @throws \Cake\Core\Exception\Exception
+     * @return array Hash with jobs as found in Configure
+     */
+    protected function getJobs()
+    {
+        $jobs = Configure::read('Gearman.Jobs');
```

Comment:
First, I intentionally created a BC breaking situation by moving the Jobs key below Gearman to prevent conflicts with other configuration settings.

Checking here for Jobs would mean doing the same in bootstrap. I have no problem with that but was hoping for a BC-breaking release and take it from there. Next release should also contain Travis and tests if this plugin is ever going to make it IMHO.